addons/shape-categories/addon.json

```
@@ -0,0 +1,21 @@
+{
+    "name": "Old Shaped Categories",

```

Minor detail, all of the other addons (with a few exceptions) don't start every word with an uppercase letter. They actually look like this:
```suggestion
    "name": "Old shaped categories",
```

---

Done with changes